config/recSlidingWindow.py

```diff
@@ -24,13 +24,40 @@
 path_to_detector = simargs.detectorPath
 winEta = simargs.winEta
 winPhi = simargs.winPhi
+winEtaSeed = int(math.floor(winEta*2./4.))
+winPhiSeed = int(math.floor(winPhi*2./4.))
+winEtaPos = int(math.floor(winEta*2./4.))
```

Comment:
then just tell me what a good choice is. or do you want to set 6 parameters in the command line? I tested this ratios with pions and now also on some 7GeV electrons, and the reconstructed clusters look good.

Reply:
The good values were the ones that are used. The sliding window algorithm is not meant ultimatelly for pions, so you cannot say that something works because it does work for pions. Moreover, as I already wrote, it does matter e.g. how the position is calculated, or in which widnow the duplicates are removed. This is something you are unable to test using the single particles. Moreover, those values we optimised with Jana for electrons long time ago. If you want to change them, implement all as parameters. If you want to use defaults for pions, do winSeed = XEta,XPhi if particle == 11 or YEta, YPhi if particle == 211 (something along those lines).